modules/extensions/src/content-types/dropdown.js

```diff
@@ -1,76 +1,8 @@
 const base = require('./_base')
```

Comment:
Also, the dropdown was moved from "extensions" to "builtin" in the latest master